platform/linux-generic/odp_ipsec.c

```diff
@@ -138,6 +138,13 @@ int odp_ipsec_capability(odp_ipsec_capability_t *capa)
 	capa->inline_ipsec_tm = ODP_SUPPORT_NO;
 
 	capa->test.sa_operations.seq_num = 1;
+
+	capa->reassembly.ip = ODP_SUPPORT_NO;
+	capa->reassembly.ipv4 = ODP_SUPPORT_NO;
+	capa->reassembly.ipv6 = ODP_SUPPORT_NO;
+	capa->reass_async_mode = ODP_SUPPORT_NO;
+	capa->reass_inline_mode = ODP_SUPPORT_NO;
```

Comment:
These are not of type odp_support_t but odp_bool_t.

Reply:
Agreed. Will fix this in v18.